sdk/modelsrepository/Azure.IoT.ModelsRepository/tests/GetModelIntegrationTests.cs

```diff
@@ -80,13 +80,12 @@ public void GetModelInvalidDtmiDependencyThrowsException()
 
         [TestCase(ModelsRepositoryTestBase.ClientType.Local, true)]
         [TestCase(ModelsRepositoryTestBase.ClientType.Local, false)]
-        [TestCase(ModelsRepositoryTestBase.ClientType.Remote, true)]
```

Comment:
Removed the Remote tests since the new test file is not available in the global DMR

Reply:
But this reduces coverage of remote, why not change how this test works or write a new test for local only?

Reply:
remote is already covered in other tests, I was thinking to add a new one, but thought it would be better to remove remote (knowing that the DMR is archived now).

If this is a blocker I can add another test instead

Reply:
> remote is already covered in other tests

The scenario this test is covering is probably not covered in other tests against remote, that's why we include the remote test cases.

> but thought it would be better to remove remote (knowing that the DMR is archived now).

This client still points to DMR by default, and the other remote tests depend on DMR.

> If this is a blocker I can add another test instead

I see JD already approved, so use your best judgement on what is right.
 

Reply:
reverted changes, and added a new integration test to verify versionless and minor versions